src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -498,6 +550,8 @@ private enum SuppressionType {
         ON,
         /** Off suppression type. */
         OFF,
+        /** Default suppression type. */
+        DEFAULT,
```

Comment:
I decided to go with `DEFAULT`. Not sure if this is the best choice, let me know what you think.

Reply:
`NEARBY` sounds good.